README.md

```diff
@@ -81,8 +74,6 @@ For interactive testing, you may instead add to `~/.mavenrc` (cf. comment in MNG
 ```sh
 export AWS_PROFILE=…
```

Comment:
Still unclear on why the profile would not be configured via GUI as well…I guess this could change if and when we introduce a GUI for credentials?

Reply:
yep, it is the idea, but first, let's have a GUI and then we add the credential configuration.